### ISSUE TITLE
[Organizer Request] Add Karin Aït-Si-Amer as co-organizer of Cloud Native Montpellier Chapter

### ISSUE THREAD

**Issue by juliencresta** [cncf/communitygroups#201]
### Name of chapter inquiring about

Cloud Native Montpellier

### Tell us about yourself. List your name, email, company, GitHub URL, LinkedIn URL, and CNCF Slack workspace name (or if you need to be invited), and a required profile link on community.cncf.io

Thanks to add : 

> Karin Aït-Si-Amer
> aitsiamer.karin@gmail.com
> https://www.linkedin.com/in/karin-a%C3%AFt-si-amer-9337511b/

to the Cloud Native Montpellier Chapter.

### Tell us why you want to be added or removed from the chapter

Karin is already apart of the team and helping us organizing events, select talks and finding host. 

@AudMonte01  

### Code of Conduct

- [x] I agree to follow the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md)

### Where organizer resides

- [ ] I confirm I am local to or live near the city of the above-proposed chapter (do not mark if not relevent to request)

### Organizers companies/affiliations

- [ ] I come from xyz company which is differing from that of the current organizers (do not mark if not relevent to request)

### Inclusive

- [ ] I agree. To assure this, I have recently completed the [Inclusive Open Source Community Orientation (LFC102) course](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/) (do not mark if not relevent to request)

### Vendor Neutral

- [ ] I have read what [vendor-nuetrality means](https://contribute.cncf.io/maintainers/community/vendor-neutrality/), and I agree the main content of our meetups will remain vendor-nuetral (do not mark if not relevent to request)

### Remain an active chapter

- [ ] I understand the activity level standards of all CNCF chapters (do not mark if not relevent to request)

### Collaboration with other chapters

- [ ] I acknowledge the CNCF right to approve additional chapters in my area based on this criteria (do not mark if not relevent to request)

### Aim to have a diverse representation of speakers in your meetups

- [ ] I will do my best to host diverse speakers throughout the year (do not mark if not relevent to request)

**Comment by AudMonte01** [comment]
Following up here, @juliencresta. Please comment with Karin's GitHub issue here. Thanks in advance.

**Comment by Kr1Kr1** [comment]
Hello!
Apologies for the delay. Here is the Credly badge confirming completion of the Open and Inclusive course by Karin Aït-Si-Amer:
🔗 https://www.credly.com/badges/f875a8e1-027b-4762-90a6-88d53d34346e/public_url

Thank you!

**Comment by AudMonte01** [comment]
Thank you, @Kr1Kr1! And do you agree to the all the above organizer terms?

**Comment by AudMonte01** [comment]
@Kr1Kr1 - following up here, as the above checkmarks are un-checked.

**Comment by Kr1Kr1** [comment]
I strongly agree with all the above organizer terms, I've quoted them since I can't edit the post from Julien

> ### Name of chapter inquiring about
> Cloud Native Montpellier
> 
> ### Tell us about yourself. List your name, email, company, GitHub URL, LinkedIn URL, and CNCF Slack workspace name (or if you need to be invited), and a required profile link on community.cncf.io
> Thanks to add :
> 
> > Karin Aït-Si-Amer
> > [aitsiamer.karin@gmail.com](mailto:aitsiamer.karin@gmail.com)
> > https://www.linkedin.com/in/karin-a%C3%AFt-si-amer-9337511b/
> 
> to the Cloud Native Montpellier Chapter.
> 
> ### Tell us why you want to be added or removed from the chapter
> Karin is already apart of the team and helping us organizing events, select talks and finding host.
> 
> [@AudMonte01](https://github.com/AudMonte01)
> 
> ### Code of Conduct
> * [x]  I agree to follow the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md)
> 
> ### Where organizer resides
> * [x]  I confirm I am local to or live near the city of the above-proposed chapter (do not mark if not relevent to request)
> 
> ### Organizers companies/affiliations
> * [x]  I come from my own company which is differing from that of the current organizers (do not mark if not relevent to request)
> 
> ### Inclusive
> * [x]  I agree. To assure this, I have recently completed the [Inclusive Open Source Community Orientation (LFC102) course](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/) (do not mark if not relevent to request)
> 
> ### Vendor Neutral
> * [x]  I have read what [vendor-nuetrality means](https://contribute.cncf.io/maintainers/community/vendor-neutrality/), and I agree the main content of our meetups will remain vendor-nuetral (do not mark if not relevent to request)
> 
> ### Remain an active chapter
> * [x]  I understand the activity level standards of all CNCF chapters (do not mark if not relevent to request)
> 
> ### Collaboration with other chapters
> * [x]  I acknowledge the CNCF right to approve additional chapters in my area based on this criteria (do not mark if not relevent to request)
> 
> ### Aim to have a diverse representation of speakers in your meetups
> * [x]  I will do my best to host diverse speakers throughout the year (do not mark if not relevent to request)

**Comment by AudMonte01** [comment]
Thank you, @Kr1Kr1 ~ I have added you to the chapter as an organizer. You should have just received an automated welcome email from me with next steps. And if you agree to the above organizer terms, please check the box upon accessing the chapter via Bevy.

Please be sure to add a profile picture to your community.cncf.io account so it may be forward-facing on the chapter page.

Should you have any questions, you can always join my office hours on Fridays. The times vary slightly: https://community.cncf.io/community-event-organizer-hub/

This issue is now closed.
